pkg/enums/enums.go

```diff
@@ -11,4 +11,6 @@ const (
 	Enforce     = "DATREE_ENFORCE"
 	Namespace   = "DATREE_NAMESPACE"
 	PodName     = "POD_NAME"
+	ActionOnFailureMonitor = "monitor"
```

Comment:
this does not belong here, create a separate enum called `ActionOnFailure` with 2 values: monitor, enfore